splink/comparison_template_library.py

```diff
@@ -216,3 +226,197 @@ def __init__(
+        - Anything else
+
+        Args:
+            col_name (str): The name of the column to compare
+            include_exact_match_level (bool, optional): If True, include an exact match
+                level for col_name. Defaults to True.
+            phonetic_col_name (str): The name of the column with phonetic reduction
+                (such as dmetaphone) of col_name. Including parameter will create
+                an exact match level for  phonetic_col_name. The phonetic column must
+                be present in the dataset to use this parameter.
+                Defaults to None
+            term_frequency_adjustments_name (bool, optional): If True, apply term
+                frequency adjustments to the exact match level for "col_name".
+                Defaults to False.
+            term_frequency_adjustments_phonetic_name (bool, optional): If True, apply
+                term frequency adjustments to the exact match level for
+                "phonetic_col_name".
+                Defaults to False.
+            levenshtein_thresholds (Union[int, list], optional): The thresholds to use
+                for levenshtein similarity level(s).
+                Defaults to []
+            jaro_winkler_thresholds (Union[int, list], optional): The thresholds to use
+                for jaro_winkler similarity level(s).
+                Defaults to [0.88]
+            jaccard_thresholds (Union[int, list], optional): The thresholds to use
+                for jaccard similarity level(s).
+                Defaults to []
+            m_probability_exact_match_name (_type_, optional): If provided, overrides
+                the default m probability for the exact match level for col_name.
+                Defaults to None.
+            m_probability_exact_match_phonetic_name (_type_, optional): If provided,
+                overrides the default m probability for the exact match level for
+                phonetic_col_name. Defaults to None.
+            m_probability_or_probabilities_lev (Union[float, list], optional):
+                _description_. If provided, overrides the default m probabilities
+                for the thresholds specified. Defaults to None.
+            m_probability_or_probabilities_datediff (Union[float, list], optional):
+                _description_. If provided, overrides the default m probabilities
+                for the thresholds specified. Defaults to None.
+            m_probability_else (_type_, optional): If provided, overrides the
+                default m probability for the 'anything else' level. Defaults to None.
+
+        Returns:
+            Comparison: A comparison that can be inclued in the Splink settings
+                dictionary.
+        """
+        # Construct Comparison
+        comparison_levels = []
+        comparison_levels.append(self._null_level(col_name))
+
+        if include_exact_match_level:
+            level_dict = self._exact_match_level(
```

Comment:
maybe rename this variable for clarity - it is a `ComparisonLevel`, not a dict

Reply:
&sim for the other sets of levels

Reply:
have renamed to `comparison_level` across this and `date_comparison`